template_content/{% if use_github_actions %}.github{% endif %}/workflows/check-python.yaml.jinja

```diff
@@ -0,0 +1,99 @@
+name: Check code base
```

Comment:
Minor: Name of this file is not quite right if you have typescript FYI

Reply:
ah of course, thanks

Reply:
Made the filename and descriptions language agnostic